lib/mpl_toolkits/tests/test_mplot3d.py

```diff
@@ -986,3 +986,14 @@ def test_quiver3D_smoke(fig_test, fig_ref):
     for fig, length in zip((fig_ref, fig_test), (1, 1.0)):
         ax = fig.gca(projection="3d")
         ax.quiver(x, y, z, u, v, w, length=length, pivot=pivot)
+
+
+@image_comparison(["minor_ticks.png"], style="default")
```

Comment:
I think we want to use `mpl20` now, don't we?

Reply:
either way works for me.